pandas/tests/io/formats/test_to_csv.py

```diff
@@ -607,3 +607,37 @@ def test_to_csv_errors(self, errors):
             ser.to_csv(path, errors=errors)
         # No use in reading back the data as it is not the same anymore
         # due to the error handling
+
+    def test_to_csv_binary_handle(self):
+        """
+        Binary file objects should work if 'mode' contains a 'b'.
+
+        GH 35058 and GH 19827
+        """
+        with tm.ensure_clean() as path:
+            with open(path, mode="w+b") as handle:
+                df = tm.makeDataFrame()
+                df.to_csv(handle, mode="w+b")
```

Comment:
can u read this back and compare the frames

Reply:
done